Sources/SwiftyCrop/View/CropView.swift

```diff
@@ -120,6 +120,14 @@ struct CropView: View {
         .background(.black)
     }
     
+    private func cropImage() -> UIImage? {
+        if maskShape == .circle {
```

Comment:
Hi @leoz I really appreciate your contribution!

It's a great idea to offer the ability to crop the image with a circle mask to an actual circular image. However, most cropping mechanisms I have used to build this library always crop the image as a square. The image is then displayed in a mask (e.g. circle). The actual image is still a square in the background, this is expected in most cases I use this library.

I think it is still a good idea to offer the ability to crop the image as a circle. I would however like to make this a configuration option when using the CropView. Could you add it into the `SwiftyCropConfiguration` e.g. as `cropImageCircular: Bool` with a default value of `false`? I will then adjust the README.

Best Regards
Ben

Reply:
Hi @benedom , thank you! 
I will do as you suggest and update the pull request.
Thank you!